### DISCUSSION TITLE
Allow users to change egg from a selection of allowed eggs in a nest

### DISCUSSION THREAD

**Discussion by moto6333** [pelican-dev/panel#205] [Ideas]
Allowing users to be able to pick an egg out of a list of allowed eggs. This could be used for games like Minecraft where allowing users to be able switch between Forge or Paper would be amazing without needing an admin to do it for them.

**Comment by TekExplorer** [comment]
Nests no longer exist. Perhaps this could be done with tags? My concern is that I don't really see how env variables carry between them.

**Comment by notAreYouScared** [comment]
We have no plans in letting the end user change eggs at this time. Might be able to be done with plugins when that system is implemented
